polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -7,6 +7,10 @@ import {useToggle} from '../../utilities/use-toggle';
 
 import {TooltipOverlay, TooltipOverlayProps} from './components';
 
+export type Width = 'default' | 'wide';
+export type Padding = 'tight' | 'loose';
+export type BorderRadius = 'none' | 'slight';
```

Comment:
We should move away from this naming convention and get these values to be as close to the tokens we use in our system.

Reply:
❤️ AHH, okie, I thought so ... but I'm struggling with the values as I only want to allow for one or the other and not all tokens? I was also struggling with `width` and `padding` one. I was thinking about doing the padding like in [AlphaCard](https://github.com/Shopify/polaris/blob/128f147d19354adbb24c48443e7c47ff6dd06c74/polaris-react/src/components/AlphaCard/AlphaCard.tsx#L18), [Bleed](https://github.com/Shopify/polaris/blob/128f147d19354adbb24c48443e7c47ff6dd06c74/polaris-react/src/components/Bleed/Bleed.tsx#L12), and [Box](https://github.com/Shopify/polaris/blob/128f147d19354adbb24c48443e7c47ff6dd06c74/polaris-react/src/components/Box/Box.tsx#L51)), but I wanted to just give **two** simple options. 

Would having something like: 
```TypeScript
export type Width = 'default' | 'wide'; 
export type Padding = 'default' | '4';
export type BorderRadius = 'default' | '2';
```

be better? 🤔

Reply:
cc @aveline @laurkim if you have any thoughts based on the layout work that is happening.

Reply:
> We should move away from this naming convention and get these values to be as close to the tokens we use in our system.

Agreed, having one off naming conventions doesn't make for a cohesive and intuitive system imo. I understand your intent @ginabak and I too would prefer a limited set of aliases (my go-to's are `small` | `medium` | `large`). However, for consistency and intuitiveness it may be worth exposing the whole token group (similar to [borderRadius](https://github.com/Shopify/polaris/blob/2bacc3bdf03d77a1f385758df2df264d0f521e6c/polaris-react/src/components/Box/Box.tsx#LL53C29-L53C29) and [padding](https://github.com/Shopify/polaris/blob/2bacc3bdf03d77a1f385758df2df264d0f521e6c/polaris-react/src/components/Box/Box.tsx#L51) in Box).

Reply:
hi @alex-page ❤️ I think both Aveline and Lo are out of office - is there anyone else we could ping? 😄

Reply:
👋 Can we do something like this?

```tsx
export type Padding = 'default' | Extract<SpacingSpaceScale, '4'>;
export type BorderRadius = Extract<ShapeBorderRadiusScale, '1' | '2'>;
```
It's awkward because you need the 'default' key since the container has different values for vertical / horizontal padding.

Or you could break down the padding into vertical / horizontal properties:
```tsx
export interface Padding {
  vertical?: Extract<SpacingSpaceScale, '1' | '2' | '4'>;
  horizontal?: Extract<SpacingSpaceScale, '1' | '2' | '4'>;
}
```

I'm worried the second would lead to more inconsistencies though so I'd opt for keeping the default key for now. I'm down to pair on this if you want a hand.

Reply:
you are all amazing!! THANK YOU ❤️